etc/ReplayOfflineConfiguration.py

```diff
@@ -280,6 +280,30 @@
                  maxMemoryperCore=2000,
                  dataset_lifetime=14*24*3600,#lifetime for container rules. Default 14 days
                  versionOverride=expressVersionOverride)
+                 
+addExpressConfig(tier0Config, "ALCAPPS",
+                 scenario=ppScenario,
+                 diskNode=["T2_CH_CERN"],
+                 data_tiers=["FEVT"],
+                 write_dqm=True,
+                 alca_producers=["PPSCalTrackBasedSel", "PromptCalibProdPPSTimingCalib", "PromptCalibProdPPSAlignment"],
```

Comment:
I might remember wrong but isn't the alcareco producer called `PPSCalMaxTracks`?

Reply:
Wasnt that for 2018 only? Maybe I remember wrong :/

Reply:
right. thanks